docs/site/Context.md

```diff
@@ -258,6 +258,27 @@ emitted for the existing binding followed by a `bind` event for the new binding.
 If a context has a parent, binding events from the parent are re-emitted on the
 context when the binding key does not exist within the current context.
 
+A context event listener should conform to the following signature:
```

Comment:
I find this new content a bit out of context. What is a "context event listener"? This concept hasn't been introduced and explained to readers yet.

Also as I commented https://github.com/strongloop/loopback-next/pull/4377#discussion_r364745654, we should explain our users what is the difference between  `ContextObserver` and `ContextEventListener` & when to use which one.